packages/mdc-segmented-button/README.md

```diff
@@ -0,0 +1,335 @@
+# Segmented Buttons
+
+[Segmented buttons](https://material.io/components/buttons#toggle-button) allow users to toggle the selected states of grouped buttons.
+
+## Using segmented buttons
+
+## Installation
+
+```
+npm install @material/segmented-button
+```
+
+## Basic Usage
+
+### HTML Structure
+
+```html
+<div class="mdc-segmented-button" role="group">
+  <button class="mdc-segmented-button__segment" aria-pressed="false">
+    <i class="material-icons mdc-segmented-button__icon">favorite</i>
+  </button>
+  <button class="mdc-segmented-button__segment" aria-pressed="false">
+    <div class="mdc-segmented-button__label">Sample Text</div>
+  </button>
+  <button class="mdc-segmented-button__segment" aria-pressed="false">
+    <i class="material-icons mdc-segmented-button__icon">favorite</i>
+    <div class="mdc-segmented-button__label">Sample Text</div>
+  </button>
+</div>
+```
+
+### Styles
+
+```scss
+@use "@material/segmented-button/segmented-button";
+@use "@material/segmented-button/segment";
+
+@include segmented-button.core-styles();
```

Comment:
Let's update this section to reflect the recent change about MDC ripple common styles.